triangulator/ear_clipping_method.py

```
@@ -1,6 +1,5 @@
-from triangulator.geometry.angles import get_polygon_internal_angle
-from triangulator.geometry.points import is_point_inside_triangle_barycentric
-
+import numpy as np
+import math
```

Ruff (0.8.2)

`math` imported but unused

Remove unused import: `math`

(F401)